Source/WebCore/platform/graphics/Image.cpp

```diff
@@ -111,6 +112,16 @@ RefPtr<Image> Image::create(ImageObserver& observer)
     return BitmapImage::create(&observer);
 }
 
+std::optional<Ref<Image>> Image::create(RefPtr<ShareableBitmap>&& bitmap)
+{
+    if (!bitmap)
+        return std::nullopt;
+    RefPtr image = bitmap->createImage();
+    if (!image)
+        return std::nullopt;
```

Comment:
I'm a little surprised we have so many ways to fail to decode instead of just making an empty image, but this does in fact preserve existing behavior.